src/dispatch/plugins/dispatch_slack/case/messages.py

```diff
@@ -316,8 +324,8 @@ def create_genai_signal_analysis_message(
     ).first()
 
     if not first_instance_id or not first_instance_signal:
-        log.warning("Unable to generate GenAI signal analysis. No signal instances found.")
```

Comment:
I'd keep the warnings so we can measure how often this happens.

Reply:
Done.